Dialyzer warn unknown

lucafavatella opened the pull request.

elvisci commented on src/ktn_meta_SUITE.erl:
```diff
@@ -57,7 +57,7 @@ dialyzer(Config) ->
   Warnings =
     case test_server:lookup_config(dialyzer_warnings, Config) of
-      undefined -> [error_handling, race_conditions, unmatched_returns];
+      undefined -> [error_handling, race_conditions, unmatched_returns, unknown];
```
Line 60 is too long:       undefined -> [error_handling, race_conditions, unmatched_returns, unknown];.

lucafavatella replied:
ok, changing this...

lucafavatella commented:
From https://github.com/inaka/sumo_rest/pull/73#issuecomment-284850381

Commit:
Add -Wunknown to default Dialyzer warnings

If considered too strict, users can override list of warnings in their test config.

lucafavatella commented:
Changed line length and force-pushed.

@elbrujohalcon It is all yours now.

elbrujohalcon commented on src/ktn_meta_SUITE.erl:
```diff
@@ -57,7 +57,10 @@ dialyzer(Config) ->
   Warnings =
     case test_server:lookup_config(dialyzer_warnings, Config) of
-      undefined -> [error_handling, race_conditions, unmatched_returns];
+      undefined -> [error_handling
```
If you're gonna use ROK-style, you have to add a space after [ here

lucafavatella replied:
addressed in force-pushed commit

elbrujohalcon commented on src/ktn_meta_SUITE.erl:
```diff
@@ -57,7 +57,10 @@ dialyzer(Config) ->
+      undefined -> [error_handling
+                   , race_conditions
+                   , unmatched_returns
+                   , unknown];
```
In ROK-style, the last ] goes below the commas in the next row.

lucafavatella commented:
@elbrujohalcon Shall be ok now.
